### ISSUE TITLE
Operations plugin does not include fields from fragments

### ISSUE THREAD

**Issue by ertrzyiks** [zhouzi/graphql-codegen-factories#30]
Hey, thanks a lot for the latest updates!

Here is my config
```yml
overwrite: true
schema: "graphql/schema.json"
documents:
  - "src/**/*.{js,jsx}"
  - ./node_modules/gatsby*/!(node_modules)/**/*.js
generates:
  ./src/types.ts:
    plugins:
      - typescript
  ./src/:
    preset: near-operation-file
    presetConfig:
      extension: .generated.ts
      baseTypesPath: types.ts
    plugins:
      - typescript-operations
      - graphql-codegen-factories/operations
```

and I installed
```
"graphql-codegen-factories": "^1.0.0-beta.2",
```

The queries that are include fragments generate the function that misses the fields from the fragment.

```gql
#import "../fragments/myData.gql"
query GetMyData {
  data {
    id
    ...MyData
  }
}
```

It generates an object with just `data: { id }` not matter what is selected by the fragment.

**Comment by zhouzi** [comment]
Isn't the query missing the fragment itself?

```graphql
query GetMyData {
  data {
    id
    ...MyData
  }
}
fragment MyData on Data {
  ...
}
```

Or perhaps you are using something that automatically loads the fragments? Does the `typescript/operations` plugin generate a type with the properties of the fragment?

**Comment by ertrzyiks** [comment]
> Does the typescript/operations plugin generate a type with the properties of the fragment?

It does, I will try to create a minimum repo where this issue occurs.

**Comment by zhouzi** [comment]
Thanks for the heads up and the reproduction @ertrzyiks 🙏 I will look into it and add a test.

**Comment by ertrzyiks** [comment]
Thanks for prompt reaction to this @zhouzi 

Please let me know if there is anything I can help with, I don't want to just throw problems at you, I can also help fixing them. I planned to work on a similar tool, but since this one exists and looks great I prefer to just contribute here instead.

**Comment by zhouzi** [comment]
No problem @ertrzyiks I am happy to help and they are interesting challenges. You are very welcome to give it a go if you feel like it though.

The current implementation of the `graphql-codegen-factories/operations` plugin is heavily inspired by `@graphql-codegen/typescript-operations`. The code loading the fragments is almost identical. So my guess is that I missed something in the way the fragments are loaded or perhaps there is a method to implement for "external fragments". It could be something else though.

Would you be interested in looking into it?

**Comment by ertrzyiks** [comment]
@zhouzi I would love to help, the investigation will take me a week or two. If you are ok with that, let me fix this problem.

**Comment by zhouzi** [comment]
Sure, go ahead @ertrzyiks and take your time 🙂

**Comment by zhouzi** [comment]
Fixed in #35 and published with `graphql-codegen-factories@1.0.0-beta.3`.
